README.md

```diff
@@ -1,4 +1,7 @@
 # P2P Task Manager
+![CI](https://github.com/PJ-ODS-2021/P2P-Task/actions/workflows/main.yaml/badge.svg)
```

Comment:
Please add a newline after the section header:
https://github.com/DavidAnson/markdownlint/blob/v0.23.1/doc/Rules.md#md022